/**:
  ros__parameters:
    lane_number: 4
    primitive_number: 6
    available_lanelet_id: [24720, 25079,  25438,  25797,
                          9213,   9688,   10163,  10638,
                          11329,  11400,  11471,  11542,
                          11679,  11692,  11705,  11718,
                          18888,  19823,  20758,  21693,
                          22792,  22799,  22806,  22813,
                          ]
    
    position:
      x: [392.613,    389.1136,   385.6138,   382.11395,
          128.4381,   128.49785,  128.55755,  128.6173,
          -357.68925, -357.7118,  -357.73435, -357.75685,
          -36.41565,  -33.64445,  -30.8711,   -28.09885,
          4.4286,     7.8059,     11.30585,   14.8058,
          160.40405,  160.3734,   160.3428,   160.3122,
        ]
        
      y: [181.0724,   181.109,    181.14555,  181.1821,
          -17.50225,  -14.00275,  -10.50325,  -7.00375,
          -16.02085,  -12.5209,   -9.02095,   -5.52105,
          -345.54245, -347.68035, -349.81545, -351.95195,
          161.1864,   161.20015,  161.21385,  161.2276,
          375.1402,   371.64035,  368.1405,   364.64065,
        ]
      z: 0.0

    orientaion:
      x: 0.0
      y: 0.0
      z: [-0.7, 1.0, 1.0, 0.45, 0.7, 0.0]
      w: [0.7, 0.0, 0.0, 0.9, 0.7, 1.0]

    desired_lane: [2, 2, 2, 2, 2, 2]

    primitive_0: [24720,  25079,  25438,  25797,
                  13685,  13694,  13703,  13712,
                  28960,  28967,  28974,  -1,
                  13807,  13826,  13845,  13864,
                  9213,   9688,   10163,  10638,
                ]

    primitive_1: [9213,   9688,   10163,  10638,
                  30753,  30760,  30767,  -1,
                  10683,  10692,  10701,  10710,
                  30122,  30129,  30136,  -1,
                  10867,  10900,  10933,  10966,
                  36008,  36095,  36182,  36269,
                  11329,  11400,  11471,  11542,
                ]

    primitive_2: [11329,  11400,  11471,  11542,
                  31852,  31871,  31890,  31909,
                  11587,  11596,  11605,  11614,
                  33887,  33894,  33901,  33908,
                  11679,  11692,  11705,  11718,
                ] 

    primitive_3: [11679,  11692,  11705,  11718,
                  34607,  34614,  34621,  34628,
                  12915,  13156,  13397,  13640,
                  1818,   1881,   1944,   2007,
                  18888,  19823,  20758,  21693,
                ]

    primitive_4: [18888,  19823,  20758,  21693,
                  28120,  28267,  28414,  28561,
                  22326,  22451,  22576,  22701,
                  31556,  31563,  31570,  31577,
                  22736,  22743,  22750,  22757,
                  30171,  30178,  30185,  30192,
                  22792,  22799,  22806,  22813,
                ] 

    primitive_5: [
                  22792,  22799,  22806,  22813,
                  30919,  30926,  30933, -1,
                  22880,  22895,  22910,  22925,
                  32823,  32974,  33125,  33276,
                  24720,  25079,  25438,  25797,
                ]

    curve_lanelet_id: [
                  # 1번째 구간
                  30753,  30760,  30767,  -1,
                  10683,  10692,  10701,  10710,
                  30122,  30129,  30136,  -1,
                  10867,  10900,  10933,  10966,
                  # 2번째 구간
                  28120,  28267,  28414,  28561,
                  22326,  22451,  22576,  22701,
                  31556,  31563,  31570,  31577,
                  22736,  22743,  22750,  22757,
                  ]